include/albatross/src/covariance_functions/radial.hpp

```diff
@@ -30,7 +30,7 @@ inline double squared_exponential_covariance(double distance,
 
 /*
  * SquaredExponential distance
- *  - c(d) = -exp((d/length_scale)^2)
+ *    covariance(d) = sigma**2 exp(-(d/length_scale)^2)
```

Comment:
nit: mixing `**` and `^`. I think carrot is more recognizable, as `**` is python/java? (even if ^ is xor in C)